website/src/components/Chat/ChatSection.tsx

```diff
@@ -1,20 +1,25 @@
 import { FormProvider, useForm } from "react-hook-form";
-import { WorkParametersInput } from "src/types/Chat";
+import { ChatConfigForm } from "src/types/Chat";
 
 import { ChatConfigDrawer } from "./ChatConfigDrawer";
 import { useChatContext } from "./ChatContext";
 import { ChatConversation } from "./ChatConversation";
 
 export const ChatSection = ({ chatId }: { chatId: string }) => {
   const { modelInfos } = useChatContext();
-  const form = useForm<WorkParametersInput>({
-    defaultValues: modelInfos[0].parameter_configs[0].work_parameters,
+
+  console.assert(modelInfos.length > 0, "No model config was found");
```

Comment:
I think we can ignore this and assume the backend always returns at least 1 config.

Reply:
Yes, this `console.assert` is there to inform other developers that we expect the array to have at least one. We know that, any new person who might look over this might not know this.